launch.bat

```diff
@@ -63,13 +63,34 @@ goto Start
         )
     )
 
+    :: Check for C++ build tools. 'cl' is the C++ compiler.
+    where cl >nul 2>&1
+    if !errorlevel! neq 0 (
+        echo "Microsoft C++ build tools not found."
+
+        where winget >nul 2>&1
+        if !errorlevel! neq 0 (
+            echo "Microsoft C++ Build Tools not found."
```

Comment:
Isn't this `Microsoft C++ Build Tools not found.` message going to be printed twice if `cl` _and_ `winget` is not installed? (note the inconsistent casing with "Build Tools" as well).